plutus-tx-plugin/test/Lift/poly.plc.golden

```diff
@@ -3,16 +3,22 @@
   [
     [
       [
-        (lam
-          Plugin_Data_Spec_Poly1_i0
-          (lam
-            Plugin_Data_Spec_Poly2_i0
+        (force
+          (delay
```

Comment:
Is it okay that `forceDelayCancel` doesn't happen in `test/Lift` tests?

Reply:
Yes: at the moment no optimization happens in `Lift`, this just makes that consistent. We probably *should* do some optimization in `Lift`, but that's a separate problem.